### PR TITLE
[13.0][MIG] project_list

### PR DESCRIPTION
**Pull request by Saran440**

### CODE REVIEW

File: project_list/__manifest__.py [hunk 1]
```diff
@@ -2,21 +2,15 @@
...
-    'installable': True,
-    'application': False,
-    'summary': 'Projects list view',
-    'depends': [
-        'project',
-    ],
-    'data': [
-        'views/project_project.xml',
-    ],
+    "name": "Projects List View",
```
Comment (alexey-pelykh) [inline project_list/__manifest__.py:5]: Useless change

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]
What's the point of "[IMP] project_list: black, isort" commit?

**Comment by yostashiro** [comment]
@alexey-pelykh Isn't this just about adopting the new style check approach?
https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97

**Review by alexey-pelykh — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
Indeed, yet probably that should be part of MIG commit then?

**Review by tarteo — COMMENTED** [review]
Isn't this functionality already in Odoo (Project > Configuration > Projects)

**Comment by alexey-pelykh** [comment]
@tarteo isn't that menu not accessible for regular users?

**Comment by Saran440** [comment]
@alexey-pelykh I read from wiki https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0
and it's suggest that 

> it is important to do all Black formatting in one commit the first time

**Comment by alexey-pelykh** [comment]
@Saran440 👍 thanks for the PR

**Comment by tarteo** [comment]
@alexey-pelykh Yes true, maybe mentioning in the readme would be nice @Saran440 .

**Review by tarteo — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Pull request merged**